src/app/contexts/EventContext/useHandlerMap/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Will store mappings between query selectors and event handler functions
+ * @param {object} _map a datastore for mapping query selectors to array of event handler functions
+ */
+export const useHandlerMap = _map => e => {
```

Comment:
there is 2 exports of the same function here too. Just make it a default export

Reply:
Fixed!